content/introduction/_index.md

```diff
@@ -96,10 +96,10 @@ The official charter for SODA Foundation under Linux Foundation can be found [he
 Please join us to collaborate, research, develop and lead for unified data framework and standards.
 Contact us through any of the following channels:
 
- - [SODA Website](https://sodafoundation.io/the-foundation/join/)  
- - [SODA Slack](https://sodafoundation.io/slack)
- - [SODA Twitter](https://twitter.com/sodafoundation)  
- - [SODA Mailing List](https://lists.sodafoundation.io)
- - [SODA LinkedIn](https://www.linkedin.com/company/sodafoundation/)
+ - <img src="https://github.com/FortAwesome/Font-Awesome/blob/6.x/svgs/brands/github.svg" width="20" height="20">&nbsp; [SODA Website](https://sodafoundation.io/the-foundation/join/)  
+ - <img src="https://github.com/FortAwesome/Font-Awesome/blob/6.x/svgs/brands/slack.svg" width="20" height="20">&nbsp; [SODA Slack](https://sodafoundation.io/slack)
+ - <img src="https://github.com/FortAwesome/Font-Awesome/blob/6.x/svgs/brands/twitter.svg" width="20" height="20">&nbsp; [SODA Twitter](https://twitter.com/sodafoundation)  
```

Comment:
Also check the Hugo [theme documentation](https://learn.netlify.app/en/cont/icons/)
You can this to insert logos. 
For inserting images you can check [this link](https://learn.netlify.app/en/cont/markdown/#images)